RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTGsfTrackVarProducer.cc

```diff
@@ -174,30 +210,23 @@ void EgammaHLTGsfTrackVarProducer::produce(edm::StreamID, edm::Event& iEvent, co
       chi2Value = 0;
       oneOverESuperMinusOneOverPValue = 0;
       oneOverESeedMinusOneOverPValue = 0;
+      fbrem = 0;
     } else {
       for (size_t trkNr = 0; trkNr < gsfTracks.size(); trkNr++) {
         GlobalPoint scPos(scRef->x(), scRef->y(), scRef->z());
 
-        GlobalPoint trackExtrapToSC;
-        {
-          auto innTSOS =
-              MultiTrajectoryStateTransform::innerStateOnSurface(*gsfTracks[trkNr], trackerGeometry, &magneticField);
-          auto posTSOS = extrapolator.extrapolate(innTSOS, scPos);
```

Comment:
Is the variable `extrapolator` (defined [here](https://github.com/cms-sw/cmssw/blob/f2d6925985a1fadedde5c21ce231b34525a3e1e9/RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTGsfTrackVarProducer.cc#L146)) still needed ?

Reply:
no, will remove